src/components/layers-panel/arcgis-import-panel/arcgis-import-panel.tsx

```diff
@@ -0,0 +1,251 @@
+      border-radius: 8px;
+    `}
+  &:hover {
+    background: ${({ theme }) => theme.colors.mainDimColor};
+    box-shadow: 0px 17px 80px rgba(0, 0, 0, 0.1);
+    border-radius: 8px;
+  }
+`;
+
+const Radio = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 44px;
+  height: 44px;
+`;
+
+const TitleCellContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  align-items: center;
+  gap: 4px;
+  cursor: pointer;
+`;
+
+const IconContainer = styled.div<{ enabled: boolean }>`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 2px;
+  width: 16px;
+  height: 16px;
+  fill: ${({ theme }) => theme.colors.buttonDimIconColor};
+  visibility: ${({ enabled }) => (enabled ? "visible" : "hidden")};
+`;
+
+export const ArcGisImportPanel = ({ onImport, onCancel }: InsertLayerProps) => {
+  const dispatch = useAppDispatch();
+  const arcGisContentArray = useAppSelector(selectArcGisContent);
+  const arcGisContentSelected = useAppSelector(selectArcGisContentSelected);
+  const sortAscending = useAppSelector(selectSortAscending);
+  const sortColumn = useAppSelector(selectSortColumn);
+  const loadingStatus = useAppSelector(selectStatus);
+
+  const isLoading = loadingStatus === "loading";
+
+  useEffect(() => {
+    dispatch(resetArcGisContentSelected());
+  }, []);
+
+  const handleImport = () => {
+    const arcGisItem = arcGisContentArray.find(
+      (item) => item.id === arcGisContentSelected
+    );
+    if (arcGisItem) {
+      onImport(arcGisItem);
+    }
+  };
+
+  const onSort = (dataColumnName: string) => {
+    if (sortColumn === dataColumnName) {
+      dispatch(setSortAscending(!sortAscending));
+    } else {
+      dispatch(setSortColumn(dataColumnName));
+    }
+  };
+
+  const formatDate = (date: number) => {
+    const formatter = new Intl.DateTimeFormat("en-US", {
+      month: "long",
+      day: "2-digit",
+      year: "numeric",
+    });
+    return formatter.format(date);
+  };
+
+  const theme = useTheme();
+  return (
+    <ModalDialog
+      title={"Select map to import"}
+      okButtonText={"Import Selected"}
+      onConfirm={handleImport}
+      onCancel={onCancel}
+    >
+      <Table>
+        <TableHeader>
+          <TableHeaderItem2>
+            <TitleCellContainer onClick={() => onSort("title")}>
+              Title
+              <IconContainer enabled={sortColumn === "title"}>
+                <SortDownIcon
+                  fill={theme.colors.buttonDimIconColor}
+                  transform={sortAscending ? "" : "rotate(180)"}
+                />
+              </IconContainer>
+            </TitleCellContainer>
+          </TableHeaderItem2>
+
+          <TableHeaderItem3>
+            <TitleCellContainer onClick={() => onSort("created")}>
+              Date
+              <IconContainer enabled={sortColumn === "created"}>
+                <SortDownIcon
```

Comment:
Seems we can use `src\components\expand-icon\expand-icon.tsx` here

Reply:
Done.